src/js/containers/awardV2/idv/ReferencedAwardsContainer.jsx

```diff
@@ -123,17 +137,21 @@ export class ReferencedAwardsContainer extends React.Component {
         });
     }
 
-    updateSort(sort, order) {
+    updateSort(newSort, newOrder) {
+        const { tableType, sort, order } = this.state;
+        sort[tableType] = newSort;
```

Comment:
`state` should not be modified directly. There is some debate about the best way to update nested state properties, I would recommend [Object.assign](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign) i.e.
```
const updatedSort = Object.assign({}, this.state.sort, {
      [this.state.tableType]: newSort
});

 this.setState({
      sort: updatedSort,
       ...
}
```